libraries/core/dora-schema.json

```diff
@@ -0,0 +1,523 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "dora-rs specification",
+  "description": "Dataflow description",
+  "type": "object",
+  "required": [
+    "nodes"
+  ],
+  "properties": {
+    "_unstable_deploy": {
```

Comment:
Do we want to have this (and other) `_unstable_*` fields in the schema? If not, we should be able to ignore them using a ` #[schemars(skip)]` attribute.

I'm also fine with keeping them in, as the field names makes it clear that they are unstable.

Reply:
Okay, in the end I have removed unstable features as to not overcrowd: autocomplete and push people on features not yet complete.